packages/react-composites/playwright.config.ts

```diff
@@ -32,7 +33,11 @@ const chromeLaunchOptions = {
   ]
 };
 
+const CI_REPORTERS: ReporterDescription[] = [['dot'], ['json', { outputFile: `${OUTPUT_DIR}/e2e-results.json` }]];
```

Comment:
For reference for others:
"Dot reporter is very concise - it only produces a single character per successful test run. It is the default on CI and useful where you don't want a lot of output."
https://playwright.dev/docs/test-reporters